src/WinRPM.jl

```diff
@@ -53,21 +53,21 @@ if isunix()
     end
 elseif iswindows()
     function download(source::AbstractString; retry=5)
-        dest = Vector{UInt16}(261)
+        ps = "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
+        tls12 = "[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12"
+        client = "New-Object System.Net.Webclient"
+        # in the following we escape ' with '' (see https://ss64.com/ps/syntax-esc.html)
+        filename = joinpath(tempdir(), split(source, "/")[end])        
+        downloadfile = "($client).DownloadFile('$(@compat replace(source, "'" => "''"))', '$(@compat replace(filename, "'" => "''"))')"        
```

Comment:
shouldn't need the at-compat macro for these, believe they're method extensions in compat rather than syntax rewrites

Reply:
Actually, the old method `replace(str, pat, r)` has been deprecated in  0.7 with `replace(s::AbstractString, pat=>r; [count::Integer])`. Therefore the `@compat` for use on both 0.6 and 1.0.

Reply:
`@compat` is only needed for syntactic changes. Else you just need `using Compat` somewhere (either in the file or once for the whole package).